apache2/persist_dbm.c

```diff
@@ -906,7 +907,9 @@ static int collections_remove_stale_ex(int db_option, modsec_rec *msr, const cha
         ag_dbm = dcfg_searchAGMDBhandler(col_name, (struct agmdb_handle_entry*)(root_dcfg->agmdb_handles));
         if(ag_dbm == NULL)
             return 1;
-        return AGMDB_removeStale(ag_dbm);
+        rc2 = AGMDB_removeStale(ag_dbm);
```

Comment:
why we use rc2 and rc for the return value, should we have more consistent coding style?